File: matiasmasso/Mat.NET/DTO/DTOContactMessage.cs
using System;

namespace DTO
{
    public class DTOContactMessage : DTOBaseGuid
    {
        public string Email { get; set; }
        public string Nom { get; set; }
        public string Location { get; set; }
        public string Text { get; set; }
        public DateTime FchCreated { get; set; }

        public DTOContactMessage() : base()
        {
        }

        public DTOContactMessage(Guid oGuid) : base(oGuid)
        {
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTOContactMessage.vb
Public Class DTOContactMessage
    Inherits DTOBaseGuid
    Property Email As String
    Property Nom As String
    Property Location As String
    Property Text As String
    Property FchCreated As DateTime

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub
End Class
